src/index.tsx

```diff
@@ -1,12 +1,22 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 
-import { HelloComponent } from './hello';
 import { setup } from './setup';
+import { startHighlight } from './highlight';
+import Toolbar from './toolbar';
+
 
 setup();
 ReactDOM.render(
-    <HelloComponent/>,
-    document.getElementById('root')
+    <div className="column">
+    <Toolbar/>

```

Comment:
I'm not sure if it's github being weird, but can you check the indentation of <Toolbar/>? It should be one level more than the surrounding \<div> tag